docs/site/docs/operate/testnet/client.md

```diff
@@ -0,0 +1,11 @@
+---
+sidebar_position: 4
+---
+
+# Run the Client
+
+This is the component that participates in network validation. Our consensus clients track delegations and stake from our AVS contracts.
+
+Similar to Ethereum, Omni validators run 2 components: our consensus client, `halo`, and an EVM execution client `geth`, `erigon`, `nethermind`, etc.
+
```

Comment:
Let's just add a comment here that this is not yet enabled (so operators aren't confused about having to run it)